frontend/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "@vue/tsconfig/tsconfig.json",
+  "compilerOptions": {
+    "noImplicitAny": false,
+    "verbatimModuleSyntax": false
+  },
```

Comment:
Do we need this file? Can we have this in the existing vite.config

Reply:
I added a test library (jest-dom), that require us to setup typescript by extending the base vue tsconfig